src/input/target/mod.rs

```diff
@@ -244,17 +244,31 @@ impl TryFrom<&str> for TargetDeviceTypeId {
 /// a composite device, and are sent to a target device to be emitted.
 pub trait TargetInputDevice {
     /// Start the DBus interface for this target device
-    fn start_dbus_interface(&mut self, dbus: Connection, path: String, client: TargetDeviceClient) {
+    fn start_dbus_interface(
+        &mut self,
+        dbus: Connection,
+        path: String,
+        client: TargetDeviceClient,
+        type_id: String,
+    ) {
         log::debug!("Starting dbus interface: {path}");
         log::trace!("Using device client: {client:?}");
         tokio::task::spawn(async move {
             let name = "Gamepad".to_string();
```

Comment:
If we pass the `TargetDeviceTypeId`, we could also use that to populate this `name` variable to something more useful. E.g.

```rust
let name = type_id.name().to_string();
```